bundles/org.openhab.binding.weathercompany/README.md

```diff
@@ -0,0 +1,349 @@
+# The Weather Company Binding
+
+Provides 5-day weather forecast and *Personal Weather Station* (PWS) current
+observations from **The Weather Company**.
+This service is available only for PWS users who upload their PWS
+weather data to WeatherUnderground.
+
+## Supported Things
+
+The following thing types are supported:
```

Comment:
This list is still lacking the technical IDs of the thing types for reference.

Reply:
Ah, ok. In all the bindings I've worked on, I don't recall ever adding the IDs of the thing types. I guess I need to update those docs, too. 😉